### PR TITLE
Help is on the way

### PR DESCRIPTION
**Pull request by linusc9516**
Current status: Mobile Unfriendly design, the rest of the design should be as requested.

I hate mobile styling aaaaaaaaaaaaa (I'll do it tmr)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,87 +1,108 @@
...
-            <li><a id="osu" href="https://osu.ppy.sh/users/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('osu')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/osu.svg"><p>osu!</p></a></li>
-            <li><a id="github" href="https://github.com/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('github')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/github.svg"><p>GitHub</p></a></li>
-          </ul>
+<body>
+    <div class="hero"></div>
+    <nav id="navbar">
+        <div class="logo">RedcXca</div>
+        <div class="nav-link-wrapper">
+            <ul class="nav-links">
+                <li><a href="./index">About</a></li>
```
Comment (RedcXca) [inline index.html]: these should have .html

File: index.html [hunk 2]
```diff
@@ -1,87 +1,108 @@
...
+            <p>So as a joke, I went to my friend's house wearing Pekora's wig and clothes. <br>I could barely stop my
+                laughter
+                as he went
+                as red as a tomato and looked at me from head to toe with a bit of drool in his mouth. <br>The way he
+                stared
+                made mde feel a
+                bit funny too, but I decided to tease him more by taking off my clothes. <br>He asked me, "Are you
+                serious?"
+                and
+                I said "Yep
```
Comment (RedcXca) [inline index.html]: LOL i feel like this is gonna get me flagged for something is there a less nsfw copypasta

File: index.html [hunk 3]
```diff
@@ -1,87 +1,108 @@
...
+                    <li>
+                        <a class="commission-btn box-bold"
+                            href="https://docs.google.com/document/d/1CELGlKdLDq_-Usue3oQfcrawi-vKOXa3SOkzTHnqB5A/edit#heading=h.9lj1hkizyon2"
+                            target="_blank">Commission Info</a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+    </header>
+    <section id="gallery-preview">
```
Comment (RedcXca) [inline index.html:79]: can you add a js function to make this scroll when the dom is loaded

File: index.html [hunk 4]
```diff
@@ -1,87 +1,108 @@
...
+                <img src="src/images/art/Wishful Wake.webp" alt="Wishful Wake Art">
+                <img src="src/images/art/osu! Winter 2023 Fanart.webp" alt="osu! Winter 2023 Fanart">
+                <img src="src/images/art/osu! Beach Episode Fanart.webp" alt="osu! Beach Episode Fanart">
+                <img src="src/images/art/osu! Halloween 2023 Fanart.webp" alt="osu! Halloween 2023 Fanart">
+                <img src="src/images/art/osu! Beatmap World Fanart.webp" alt="osu! Beatmap World Fanart">
+        </div>
+        <button class="arrow" id="right-arrow">&#9654;</button>
+        </div>
+        <div class="button-container">
+            <a href="./illustrations" class="view-gallery">View Gallery</a>
```
Comment (RedcXca) [inline index.html]: .html

File: illustrations.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        crossorigin="anonymous"></script>
+    <link href="./src/css/illustrations.css" rel="stylesheet" />
+    <link href="./src/css/shared.css" rel="stylesheet" />
+    <style>
+        @import url("https://fonts.googleapis.com/css2?family=Dosis:wght@200..800&display=swap");
+    </style>
+</head>
+
+<body>
+    <div class="hero"></div>
```
Comment (RedcXca) [inline illustrations.html:22]: hero should probably go under navbar to not block it (or maybe its just the transparent-nav class)

File: index.html [hunk 6]
```diff
@@ -1,87 +1,108 @@
...
+        </div>
+    </nav>
+    <header>
+        <img src="src/images/早安.png" class="circle-image">
+        <section>
+            <h1 class="title">Max Lu/RedcXca</h1>
+            <div class="external-links">
+                <ul>
+                    <li><a id="osu" href="https://osu.ppy.sh/users/RedcXca" target="_blank"
+                            onmouseover="ExternalLinksEnter('osu')" onmouseleave="ExternalLinksLeave()"><img
```
Comment (RedcXca) [inline index.html:52]: invert the colours of this osu svg ;w; also maybe move the buttons a bit closer

File: projects.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+                <li><a href="./index">About</a></li>
+                <li><a href="./projects">Projects</a></li>
+                <li><a href="./illustrations">Illustrations</a></li>
+                <li><a href="./osu">osu!</a></li>
+            </ul>
+        </div>
+    </nav>
+    <div class="project-container">
+        <div class="project-card">
+            <!-- <a href="https://github.com/RedcXca/girls/kissing"> -->
```
Comment (RedcXca) [inline projects.html]: no girls kissing this comment is a lie.........

**Review by RedcXca — CHANGES_REQUESTED** [review]
holy shit this looks so good, just a few small nits but most look really good, i think the exclusion for the header fits super well

-can scroll bar be dark themed 
-the hrefs should be index.html instead of index, for all the navbars and the "view gallery" on the home page
-style the text to not show hyperlink line when href of girls kissing is uncommented

File: index.html [hunk 8]
```diff
@@ -1,87 +1,108 @@
...
+                peko."<br><br> (I won't continue further)</p>
+        </div>
     </section>
-
     <footer>
-      <p class="large-footer-text"><b>ART & DESIGN © 2024 BY REDCXCA. ALL RIGHTS RESERVED.</b></p>
-      <p class="small-footer-text">And some more small text probably no one is going to see.</p>
-      <p class="small-footer-text bottom-footer-text">This entire site is like 50 lines of HTML I wrote for practice.</p>
+        <p>
+            Contact: <a href=mailto:maxzylu@gmail.com">maxzylu@gmail.com</a>
```
Comment (RedcXca) [inline index.html]: shouldn't have the "

File: index.html (hunk 1 above)
Comment (linusc9516) [reply index.html]: With a .htaccess file this won't really matter since its the same thing with or without .html suffix

File: index.html (hunk 6 above)
Comment (linusc9516) [reply index.html:52]: image
Edited the SVG to be white.

Comment (linusc9516) [reply index.html:52]: image
Made the buttons closer and align.
Don't ask why its decimals I did a bit of calculation with the rem ;w; should align on pc displays

**Comment by linusc9516** [comment]
> holy shit this looks so good, just a few small nits but most look really good, i think the exclusion for the header fits super well
> 
> -can scroll bar be dark themed
> -the hrefs should be index.html instead of index, for all the navbars and the "view gallery" on the home page
> -style the text to not show hyperlink line when href of girls kissing is uncommented

https://caniuse.com/css-scrollbar should work on both Chrome and IE. Edge and Mozilla are pretty much impossible unless using jQuery or simplebar, but seems a very small issue tbf. 
Styling code borrowed from this [codepen](https://codepen.io/subfauna/pen/jOGgrr) implementation that I really like.

hrefs are fixed, probably works with or without the .html but yeah better to put the suffix than not to for clarity's sake.

image
Fixed the hyperlink line.

Added auto-scroll on dom content loaded. Also added an infinite scrolling gallery because auto-scroll breaks without one :3

Btw changed the copypasta to another one thats more sfw haha

File: illustrations.html (hunk 5 above)
Comment (RedcXca) [reply illustrations.html:22]: okay its just the transparent-nav class (i forgot why it was there) just take it out for now

**Comment by RedcXca** [comment]
should be last review now, scroll is a little bit bugged, i think the ts might be missing imports(?)
image

remove transparent-nav from illustrations, add title text to projects, and should be good 🌮

**Review by RedcXca — CHANGES_REQUESTED** [review]
should be last review now, scroll is a little bit bugged, i think the ts might be missing imports(?) also sent a video on discord of how it acts on my end
image

remove transparent-nav from illustrations, add title text to projects, and should be good 🌮

**Pull request merged**